Mathlib/Topology/Algebra/Module/Alternating/Basic.lean

```diff
@@ -332,34 +332,49 @@ theorem _root_.ContinuousLinearMap.compContinuousAlternatingMap_coe (g : N →L[
     (f : M [⋀^ι]→L[R] N) : ⇑(g.compContinuousAlternatingMap f) = g ∘ f :=
   rfl
 
-/-- A continuous linear equivalence of domains defines an equivalence between continuous
-alternating maps. -/
-def _root_.ContinuousLinearEquiv.continuousAlternatingMapComp (e : M ≃L[R] M') :
+/-- A continuous linear equivalence of domains
+defines an equivalence between continuous alternating maps. -/
+@[simps -fullyApplied apply]
+def _root_.ContinuousLinearEquiv.continuousAlternatingMapCongrLeftEquiv (e : M ≃L[R] M') :
```

Comment:
Should this be `ContinuousLinearEquiv.continuousAlternatingMapCongrLeftEquiv` or `Equiv.continuousAlternatingMapCongrLeft`?